src/main/java/org/sagebionetworks/web/client/widget/SelectionToolbarPresenter.java

```diff
@@ -0,0 +1,114 @@
+package org.sagebionetworks.web.client.widget;
```

Comment:
Refactor so that this is a helper widget (encapsulating selection toolbar presenter logic, but not imposing hierarchy).